src/coreclr/jit/emitarm64.cpp

```diff
@@ -13849,6 +13918,20 @@ void emitter::emitDispVectorElemList(
     }
 }
 
+//------------------------------------------------------------------------
+// emitDispPredicateReg: Display a predicate register name with with an arrangement suffix
+//
+void emitter::emitDispPredicateReg(regNumber reg, insOpts opt, bool addComma)
+{
+    // TODO-SVE: Fix once we add predicate registers
+    assert(isPredicateRegister(reg));
+    printf(emitVectorRegName(reg));
```

Comment:
Seems like we need a `emitPredicateRegName` than handles the "fake" predicate registers but still outputs `p0` through `p15`

Reply:
Kunal has a temporary implementation for this in #95016. As an aside, there seems to be potential for circular dependencies between these two PRs. Should I wait for Kunal's to be merged in first? I can then update my implementation with the new methods #95016 adds in (like `emitPredicateRegName`) and also update the predicate register TODOs to use the aliases you suggested above.